loops.js

```diff
@@ -11,8 +11,10 @@
 // Dit heb je nog niet geleerd, maar bekijk hiervoor dit MDN-artikel eens: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat
 // ==========================================
 
-
-
+const star = '*';
+for (let i = 0; i < 6; i++) {
+    console.log(`${star.repeat(i)}`)
+}
```

Comment:
Kan korter: `for (let i = 0; i < 5; i++) {
  console.log('*'.repeat(i + 1));
}`